getting-started-guides/on-prem-cluster/standalone-python.md

```diff
@@ -83,7 +83,7 @@ Variables required to run spark-submit command:
 # path to xgboost4j_spark/libs
 export LIBS_PATH=/home/xgboost4j_spark/lib
 
-# additional jars for XGBoost4J example
+# additional jars for XGBoost4J example, use corresponding cudf jar accoding to your cuda runtime version.
```

Comment:
typo according

Reply:
This part has been removed as jars are moved to `$SPARK_HOME/jars` . So no extra `--jars` are required here.